janusgraph-core/src/main/java/org/janusgraph/graphdb/transaction/StandardJanusGraphTx.java

```diff
@@ -1258,6 +1258,7 @@ else if (query.getResultType() == ElementCategory.VERTEX) {
             Iterator<JanusGraphElement> iter;
             if (!indexQuery.isEmpty()) {
                 List<QueryUtil.IndexCall<Object>> retrievals = new ArrayList<QueryUtil.IndexCall<Object>>();
+                //we leave 1st index for streaming, and prepare the rest for reading in full and intersecting
```

Comment:
Add a space after comment use 3rd person
// Leave first index for streaming and prepare subsequent indices for intersecting and lookup